src/app/dao/FirebaseFirestoreDAO.ts

```diff
@@ -17,7 +22,11 @@ export interface IFirebaseFirestoreDAO<T extends FirebaseJSONObject> {
 
     set(id: string, element: T): Promise<void>;
 
-    getObsById(id: string): Observable<{id: string, doc: T}>;
+    /**
+     * Observes a specific document given its id.
+     * The observable gives an optional, set to empty when the document is deleted
+     */
```

Comment:
And when calling it with an id of a non existing document it gives empty() as well ?

Reply:
j'imagine mais pas sûr !

Reply:
ça vaudrait la peine de checker ^^ ?

Reply:
confirmé + ajouté au commentaire